lib/ui/LandingPage.dart

```diff
@@ -134,7 +134,8 @@ class LandingPage extends StatelessWidget {
 
   @override
```

Comment:
That has been done also in https://github.com/farmsmart/farmsmart-flutter/pull/109, isn't it? I think that this class should be removed from this PR.

Reply:
rebased, this should be removed from the PR now.